[ 2023-04-23 16:59:34,007 ] root - INFO - Entered the start_data_ingestion method of TrainPipeline class
[ 2023-04-23 16:59:34,007 ] root - INFO - Starting data ingestion
[ 2023-04-23 16:59:34,027 ] root - INFO - Consider Local csv file
[ 2023-04-23 16:59:35,114 ] root - INFO - Shape of dataframe: (36188, 171)
[ 2023-04-23 16:59:35,196 ] root - INFO - Shape of dataframe: (36188, 164)
[ 2023-04-23 16:59:35,196 ] root - INFO - Saving exported data into feature store file path: artifact/04_23_2023_16_59_33/data_ingestion/feature_store/sensor.csv
[ 2023-04-23 16:59:36,262 ] root - INFO - Shape of dataframe: (36188, 164)
[ 2023-04-23 16:59:36,262 ] root - INFO - Starting the train and test splittings
[ 2023-04-23 16:59:38,186 ] root - INFO - Data ingestion completed and artifact: DataIngestionArtifact(trained_file_path='artifact/04_23_2023_16_59_33/data_ingestion/ingested/train.csv', test_file_path='artifact/04_23_2023_16_59_33/data_ingestion/ingested/test.csv')
[ 2023-04-23 16:59:39,456 ] root - INFO - Missing numerical column: []
[ 2023-04-23 16:59:39,456 ] root - INFO - Valid Numerical Column : True
[ 2023-04-23 16:59:39,457 ] root - INFO - Missing numerical column: []
[ 2023-04-23 16:59:39,457 ] root - INFO - Required number of columns: 164
[ 2023-04-23 16:59:39,457 ] root - INFO - Data frame has columns: 164
[ 2023-04-23 16:59:39,457 ] root - INFO - Required number of columns: 164
[ 2023-04-23 16:59:39,457 ] root - INFO - Data frame has columns: 164
[ 2023-04-23 16:59:44,600 ] root - INFO - Data validation artifact: DataValidationArtifact(validation_status=False, valid_train_file_path='artifact/04_23_2023_16_59_33/data_ingestion/ingested/train.csv', valid_test_file_path='artifact/04_23_2023_16_59_33/data_ingestion/ingested/test.csv', invalid_train_file_path=None, invalid_test_file_path=None, drift_report_file_path='artifact/04_23_2023_16_59_33/data_validation/drift_report/report.yaml')
[ 2023-04-23 16:59:46,057 ] root - INFO - Error occurred python script name [main.py] line number [8] error message [Error occurred python script name [/home/suyodhan/Documents/Data Science Project/current_project/Heyyyyyy/sensor/pipeline/training_pipeline.py] line number [82] error message [Error occurred python script name [/home/suyodhan/Documents/Data Science Project/current_project/Heyyyyyy/sensor/pipeline/training_pipeline.py] line number [60] error message [Error occurred python script name [/home/suyodhan/Documents/Data Science Project/current_project/Heyyyyyy/sensor/components/data_transformation.py] line number [88] error message ['numpy.ndarray' object has no attribute 'replace']]]]
